staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -116,8 +119,51 @@ service RuntimeService {
     rpc Status(StatusRequest) returns (StatusResponse) {}
 }
 
+// GetEventsFilter is used to filter a list of events.
+// All those fields are combined with 'AND'
+message GetEventsFilter {
+    // ID of the container, sandbox or image.
+    string id = 1;
+    // LabelSelector to select matches.
+    // Only api.MatchLabels is supported for now and the requirements
+    // are ANDed. MatchExpressions is not supported yet.
+    map<string, string> label_selector = 3;
```

Comment:
if both id and label_selector are specified in the request, which one will take priority?

Reply:
IMO, `id` should take the priority since user has explicitly expressed the exact filtration criteria by setting the `id`. `label_selector` is much broader lookup than `id`. When the user specifies `id`, we can assume that they are interested only in the events of that particular container or image. 

On the other hand, we can also do `id` AND `label_selector` in case both have been specified by the user.

Reply:
Yeah I am thinking that `id` and `label_selector` should be mutually exclusive, i.e. users should only specify one of them, or none. If both are specified, then `id` takes precedence over `label_selector`. Does that sound good?

Reply:
Yes, sounds good. Thanks.

Reply:
how do other parts of the k8s API handle such situations? I would expect all filters to be AND'ed, but it's this defined somewhere? The use case I would see (albeit this being a rare one I think) is to make sure it's not some other like named pod, but one that satisfies additional criteria. Just my .02.

Reply:
Thanks @thediveo, indeed the other parts of that proto file use `AND`, so it makes sense for us to stay consistent. e.g. https://github.com/kubernetes/kubernetes/blob/b08ab7c027f16da257ac37e486b9caa3df336676/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L582-L593